test/verify/check-connection

```diff
@@ -107,7 +107,7 @@ class TestConnection(testlib.MachineCase):
         # cookie should not be marked as secure, it's not https
         cookie = b.cookie("cockpit")
         self.assertTrue(cookie["httpOnly"])
-        self.assertEqual(cookie["sameSite"], "Strict")
```

Comment:
Do you plan to firm this up again in the other direction after we land #20832?

Reply:
We could, but it doesn't really matter IMHO -- this is "strict" enough (haha)

Reply:
Sure.  I just like to avoid carrying these small bits of cruft that end up having no purpose anymore.  I guess in the unlikely event that it bothers someone, they can look it `git blame` and follow the trail until they find this discussion ;)

Reply:
Added to follow-up list in #20832

Reply:
#20832 now makes this strict again.